src/components/sideNavigation/tests/SideNavigationLoadingItem.spec.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
```

Comment:
same comment as for `SideNavigation.spec.tsx` check that your view renders without error before anything else